### PR TITLE
Rsp 1888 cookie compliance

### PR DESCRIPTION
**Pull request by mgoldson**
## Description
Implements cookie compliance based on www.gov.uk website

JIRA: https://jira.dvsacloud.uk/browse/RSP-1888

## Key changes

- Adds a cookie banner with options to accept all cookies or set preferences
- Adds a Cookie Preferences page allowing user to set their cookie preferences
- Adds a confirmation notification when the user updates their preferences on the Cookie Preferences page
- Adds a link in the footer to the Cookie Preferences page
- Adds Cypress tests (designed to be run locally only)

DVSA_RSP_Cookie_Banner
DVSA_RSP_Cookie_Details
DVSA_RSP_Cookie_Preferences

### CODE REVIEW

**Comment by johnmorleybjss** [comment]
Any chance of a screenshot please @mgoldon?

File: src/server/controllers/cookieDetails.controller.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import config from '../config';
+
+// Index Route
+export const index = (req, res) => {
+  const language = req.i18n_lang;
+  if (config.isDevelopment()) {
```
Comment (johnmorleybjss) [inline src/server/controllers/cookieDetails.controller.js]: Seems like the end result of whether these conditions are true or false is the same, could we simplify to just call the render in any case?

File: src/server/i18n/cy.json [hunk 2]
```diff
@@ -170,5 +178,64 @@
     "pending_notice": "Mae taliad yn yr arfaeth ar gyfer y cod talu hwn ac nid yw'r botwm hwn ar gael ar hyn o bryd. Arhoswch hyd at MINUTES munud i’r taliad gael ei brosesu neu efallai y codir tâl arnoch ddwywaith.",
     "go_back": "Dychwelyd",
     "pay_button": "Parhau"
+  },
+  "cookie_preferences_page": {
+    "title": "Cookies on Roadside Payments service",
```
Comment (johnmorleybjss) [inline src/server/i18n/cy.json]: No translations for this page yet?

Comment (mgoldson) [reply src/server/i18n/cy.json]: No translations as yet. They may be introduced at a later date.

**Pull request merged**